lib/active_utils/network_connection_retries.rb

```diff
@@ -10,18 +10,23 @@ module NetworkConnectionRetries
       OpenSSL::SSL::SSLError => "The SSL connection to the remote server could not be established"
     }
 
+    DEFAUT_RETRY_ERRORS = {
+      Errno::ECONNREFUSED => "The remote server refused the connection"
+    }
+
     def self.included(base)
       base.send(:attr_accessor, :retry_safe)
     end
 
     def retry_exceptions(options={})
       connection_errors = DEFAULT_CONNECTION_ERRORS.merge(options[:connection_exceptions] || {})
+      retry_errors = DEFAUT_RETRY_ERRORS.merge(options[:retryable_exceptions] || {})
 
       retry_network_exceptions(options) do
         begin
           yield
-        rescue Errno::ECONNREFUSED => e
-          raise ActiveUtils::RetriableConnectionError, "The remote server refused the connection"
+        rescue *retry_errors.keys => e
+          raise ActiveUtils::RetriableConnectionError, derived_error_message(retry_errors, e.class)
```

Comment:
If we're using `Retriable` down here, we should probably use the same spelling up there, though.

Reply:
Yeah, good point, I'll keep it the same even if it's technically incorrect @joshmcclain